docs/contributors/modules/docs-style/pages/headings.adoc

```diff
@@ -19,3 +19,5 @@
 * When referencing headings in prose, use the full heading with no quotes, and use title case.
 ** **Good:** For more information, see the https://circleci.com/docs/hello-world/[Hello World] guide. +
```

Comment:
This is markdown-ish syntax. I realise the point here is about the text, but I think we should still use a proper AsciiDoc syntax. 🤔